reanahub/reana
# benchmark: split script into files and add to entrypoints

closes #553 

How to test:

No new features were added to the script so testing is mainly about checking if splitting and entry points do not break existing functionality.

Steps:

1. Install `reana-benchmark` via `setup.py` with `pip install -e .` (inside `reana/` repository with this branch)

2. Run some small amount to workflow to test if all commands are available and work as expected. Example (inside, for example, `helloworld` demo):

```console
$ reana-benchmark submit -w test -n 1-3
$ reana-benchmark start -w test -n 1-3
$ reana-benchmark collect -w test
$ reana-benchmark analyze -w test -n 1-3  # -t option
```

and

```console
$ reana-benchmark launch -w test-2 -n 1-3
$ reana-benchmark collect -w test-2
$ reana-benchmark analyze -w test-2 -n 1-3
```
